### ISSUE TITLE
Console Cache List command doesn't work with RedisCacheDriver

### ISSUE THREAD

**Issue by nupplaphil** [friendica/friendica#5847]
### Expected behavior

- executing `php bin/console.php cache list`
- A list with all keys returns (at least with no key)

### Actual behavior

- executing `php bin/console.php cache list`
```
Warning: array_walk() expects parameter 1 to be array, null given in /var/www/html/src/Core/Cache.php on line 65

Warning: sort() expects parameter 1 to be array, null given in /var/www/html/src/Core/Cache.php on line 67
Listing all cache keys:

Warning: Invalid argument supplied for foreach() in /var/www/html/src/Core/Console/Cache.php on line 117
0 keys found
```

### Steps to reproduce the problem

- executing `php bin/console.php cache list`

### Friendica version you encountered the problem

`2018.12-dev` (branch `develop`)

### Friendica source (git, zip)

git

### PHP version

```
PHP 7.1.22 (cli) (built: Sep 15 2018 04:13:44) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.1.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.1.22, Copyright (c) 1999-2018, by Zend Technologies
```

### SQL version

```
mysql  Ver 15.1 Distrib 10.3.10-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2
```

**Comment by nupplaphil** [comment]
Occurs because of using `Redis` as Cache Driver and Null-Exception at `array_walk`

**Comment by MrPetovan** [comment]
Closed via #5848
